tests/migration/reset_scc_addons_values.pm

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Swap the values of SCC_ADDONS before migration.
+#
+# In migration maintenance updates tests, SCC_ADDONS is used for both
+# registering original system and adding target system modules maintenance
+# updates repos. But for SLE 12 and SLE 15, the modules name are different.
+# So swap SCC_ADDONS values to upgraded SLE 15 modules before migration.
+# And re-create incidents repos based on the swapped SCC_ADDONS values.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    set_var('SCC_ADDONS', get_var('SCC_ADDONS_2'));
```

Comment:
It's better to add comment for 'SCC_ADDONS_2'

Reply:
we should not document that variable, neither in the code above this line, neither in the file with the other variables, because we don't want that variable to be used by anyone else except this module. I will comment on the renaming of this module and description above.

Reply:
done, thanks.